pages/content/releases/release-0.7.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Release 0.7"
+weight: 997
+type: docs
+aliases:
+- /release-0.7.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Apache Paimon 0.7 Available
+
+February 29, 2024 - Paimon Community (dev@paimon.apache.org)
+
+Apache Paimon PPMC has officially released Apache Paimon 0.7.0-incubating version. A total of 34 people contributed to
+this version and completed over 300 Commits. Thank you to all contributors for their support!
+
+In this version, we mainly focus on enhancement and optimization for the existing features. For more details, please 
+refer to the following content below.
+
+## Flink
+
+### Lookup Join
+
+1. Fix bug that lookup join cannot handle sequence field of dim table.
+2. Introduced primary key partial lookup based on Paimon hash lookup for lookup join. 
+3. Use parallel reading and bulk loading to speed up initial data loading of dim table.
```

Comment:
Should we mention the new lookup query service here ?

Reply:
Not work for 0.7...

Reply:
Oh..